src/chainlit/socket.py

```diff
@@ -72,10 +84,13 @@ def ask_user_fn(data, timeout):
 
     user_env = environ.get("HTTP_USER_ENV")
     authorization = environ.get("HTTP_AUTHORIZATION")
+    chainlit_headers = load_chainlit_headers(environ.get("HTTP_COOKIE"))
```

Comment:
what is this black magic? who's modifying our process environment to put HTTP stuff into it? 😦

Reply:
environ is not the python process environ, it is coming from the socket io connect callback.

Reply:
mdr